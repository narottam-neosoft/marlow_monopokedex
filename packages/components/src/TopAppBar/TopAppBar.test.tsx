import React from 'react';
import { render, screen } from '@testing-library/react';
import {TopAppBar} from './TopAppBar';

describe('TopAppBar', () => {
  const mockProps = {
    name: 'pokedex',
  };

  it('renders the app bar with correct props', () => {
    const {container} = render(<TopAppBar {...mockProps} />);

    expect(screen.getByText('pokedex')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
