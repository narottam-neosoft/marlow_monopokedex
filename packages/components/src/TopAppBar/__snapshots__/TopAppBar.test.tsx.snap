// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopAppBar renders the app bar with correct props 1`] = `
<div>
  <div
    class="MuiBox-root css-i9gxme"
  >
    <header
      class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
    >
      <div
        class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
      >
        <div
          class="MuiTypography-root MuiTypography-h6 css-1juivf6-MuiTypography-root"
        >
          pokedex
        </div>
      </div>
    </header>
  </div>
</div>
`;
